src/server/index.test.ts

```diff
@@ -10,10 +10,6 @@ import * as fixtures from '~/test/fixtures/index.js'
 
 jest.mock('~/src/server/plugins/engine/services/formsService.js')
 
-const okStatusCode = 200
-const redirectStatusCode = 302
-const notFoundStatusCode = 404
```

Comment:
@colinrotherham out of interest, why did you remove this? Seems to make sense to keep it DRY?

Reply:
Sorry ignore my question. I saw your comment about Sonar being the culprit.

Reply:
Yeah agreed. We should swap them all for `StatusCodes` at some point